### PR TITLE
Reformatted with `black`.  Changed np.int (deprecated) to int.

### PR DESCRIPTION
**Pull request by dmopalmer**
Cleaned up the code by:
1. Reformatting the .py files with `black`.
2. Replacing np.int (which is deprecated) with int

The `.ipynb` Jupyter Notebook files were not reformatted.

This is to address #13 and a secondary aspect of  #12.

### CODE REVIEW

**Comment by parsotat** [comment]
There's a lot of changes in one commit here. For ease of review can you break this formatting change into different commits where black is run on one file at a time and then of course a final commit where np.int is converted to int?

**Comment by dmopalmer** [comment]
I can do that, but I don't know how much it will ease review.  Do you want me to push each time or only at the end.

**Comment by parsotat** [comment]
pushing each time would be great so I can just go through the files as you get them done

**Comment by dmopalmer** [comment]
I just did 5, an dpushed, so I will push after each one from now on.

**Comment by parsotat** [comment]
I would say that we should format all the .py files, including those in the notebook directory

**Comment by dmopalmer** [comment]
YOu pulled while I was still pushing.


Should I also reformat the `.ipynb` files?

File: batanalysis/bat_survey.py [hunk 1]
```diff
@@ -832,116 +984,222 @@ def calculate_pha(self, id_list, output_dir=None, calc_upper_lim=False, bkg_nsig
...
-                        print('check = ', check)
-                        print('Found more than one matched time, please double check the time interval.')
-                        print('This method does not add up the counts for more than once time intervals.')
+                    if check > 1:
+                        print("check = ", check)
+                        print(
+                            "Found more than one matched time, please double check the time interval."
+                        )
+                        print(
+                            "This method does not add up the counts for more than once time intervals."
```
Comment (parsotat) [inline batanalysis/bat_survey.py:1185]: Can we correct this type that I made a while ago? once->one

File: batanalysis/mosaic.py [hunk 2]
```diff
@@ -29,33 +29,35 @@
...
+# also information to create the skygrids if the user wants
+_gcenters = np.array(
+    [
+        [90, 0],  # Galactic equatorial belt 1
+        [0, 0],  # Galactic equatorial belt 2
+        [-90, 0],  # Galactic equatorial belt 3
+        [180, 0],  # Galactic equatorial belt 4
+        [0, 90],  # North galactic polar cap
+        [0, -90],
+    ],  # South galactic polar cap
```
Comment (parsotat) [inline batanalysis/mosaic.py:53]: This comment should be next to [0,-90], similar to those above

File: notebooks/trial_NGC2992.py [hunk 3]
```diff
@@ -14,78 +14,126 @@
...
+)
 
-queryargs = dict(time="2004-12-15 .. 2005-12-16", fields='All', resultmax=0)
+queryargs = dict(time="2004-12-15 .. 2005-12-16", fields="All", resultmax=0)
 table_everything = ba.from_heasarc(**queryargs)
 
-minexposure = 1000     # cm^2 after cos adjust
-exposures = np.array([object_batsource.exposure(ra=row['RA'], dec=row['DEC'], roll=row['ROLL_ANGLE'])[0] for row in table_everything])
+minexposure = 1000  # cm^2 after cos adjust
+exposures = np.array(
```
Comment (parsotat) [inline notebooks/trial_NGC2992.py:27]: This change here is more confusing in my opinion, especially for this example python script.

File: notebooks/trial_detection_Crab.py [hunk 4]
```diff
@@ -15,123 +15,207 @@
...
-minexposure = 1000     # cm^2 after cos adjust
-
-#calculate the exposure with partial coding
-exposures = np.array([object_batsource.exposure(ra=row['RA'], dec=row['DEC'], roll=row['ROLL_ANGLE'])[0] for row in table_everything])
-
-#select the observations that have greater than the minimum desired exposure
+minexposure = 1000  # cm^2 after cos adjust
+
+# calculate the exposure with partial coding
+exposures = np.array(
```
Comment (parsotat) [inline notebooks/trial_detection_Crab.py:35]: This change is also not great in my opinion for this example notebook

File: notebooks/trial_nondetection_MAXI.py [hunk 5]
```diff
@@ -14,74 +14,124 @@
...
 object_location = swiftbat.simbadlocation(object_name)
-object_batsource = swiftbat.source(ra=object_location[0], dec=object_location[1], name=object_name)
+object_batsource = swiftbat.source(
+    ra=object_location[0], dec=object_location[1], name=object_name
+)
 table_everything = ba.from_heasarc(**queryargs)
-minexposure = 1000     # cm^2 after cos adjust
-exposures = np.array([object_batsource.exposure(ra=row['RA'], dec=row['DEC'], roll=row['ROLL_ANGLE'])[0] for row in table_everything])
+minexposure = 1000  # cm^2 after cos adjust
+exposures = np.array(
```
Comment (parsotat) [inline notebooks/trial_nondetection_MAXI.py:25]: This is a similar comment as the other example files

**Comment by parsotat** [comment]
with the exception of the 3 comments on the example files the other reformats look good to me

File: notebooks/trial_NGC2992.py (hunk 3 above)
Comment (dmopalmer) [reply notebooks/trial_NGC2992.py:27]: What do you think of the format
```
exposures = np.array(
    [object_batsource.exposure(ra=row["RA"],
                               dec=row["DEC"], 
                               roll=row["ROLL_ANGLE"])[0]
        for row in table_everything
    ])
```
?  (For all three files).  I think it improves the clarity, even if it lacks the purity of letting `black` format everything.

**Comment by parsotat** [comment]
I think that looks great to me. Thanks David. On Nov 13, 2023, at 6:17 PM, David Palmer ***@***.***> wrote:﻿
@dmopalmer commented on this pull request.



In notebooks/trial_NGC2992.py:
>  table_everything = ba.from_heasarc(**queryargs)
 
-minexposure = 1000     # cm^2 after cos adjust
-exposures = np.array([object_batsource.exposure(ra=row['RA'], dec=row['DEC'], roll=row['ROLL_ANGLE'])[0] for row in table_everything])
+minexposure = 1000  # cm^2 after cos adjust
+exposures = np.array(

What do you think of the format
exposures = np.array(
    [object_batsource.exposure(ra=row["RA"],
                               dec=row["DEC"], 
                               roll=row["ROLL_ANGLE"])[0]
        for row in table_everything
    ])

?  (For all three files).  I think it improves the clarity, even if it lacks the purity of letting black format everything.

**Comment by dmopalmer** [comment]
Last push with those reformat.  Ready to go.

**Comment by parsotat** [comment]
This looks good to me. I will merge now.

**Pull request merged**